README.md

```diff
@@ -79,6 +79,65 @@ cd corral
 
 You can make a debug build with `.\make.ps1 build -Config Debug`
 
+## Getting started using Corral
+After instllation, make sure Corral's current path is also set in your PATH environment variable. This will make it easier by not entering the full path each time when you use one of Corral commands in your projects.
```

Comment:
It's not clear how or where, Corral would be from this.

I think it is safe to assume that Corral was installed via ponyup as that is the instructions we give.

Do you believe that someone will notice the path from the Installation instructions? I think if this is important, then adding `~/.pony/ponyup/bin` to the $PATH should also be highlighted in the Installation section.

Reply:
I've build corral from the source. I didn't have ponyup on my machine and I couldn't build ponyup on my machine because I am running windows machine and ponyup doesn't have a powershell script. Which is why I mentioned that you need to put the corral executable in one of your PATH environment variable's path.

Do you think it will be a good idea to do the following 

- In Installation section mention that you should add `~/.pony/ponyup/bin` to the $PATH
- In Building From Source section mention to add `build/release` path in $PATH environment variable
- Get rid of  "After installation, ..." paragraph which is in Getting started using corral section